book/src/ui-installation.md

```diff
@@ -1,113 +1,73 @@
 # 📦 Installation
 
-Siren runs on Linux, MacOS and Windows.
+Siren supports any operating system that supports container runtimes and/or NodeJS 18, this includes Linux, MacOS, and Windows. The recommended way of running Siren is by launching the [docker container](https://hub.docker.com/r/sigp/siren) , but running the application directly is also possible.
 
 ## Version Requirement
 
-The Siren app requires Lighthouse v3.5.1 or higher to function properly. These versions can be found on the [releases](https://github.com/sigp/lighthouse/releases) page of the Lighthouse repository.
+To ensure proper functionality, the Siren app requires Lighthouse v4.3.0 or higher. You can find these versions on the [releases](https://github.com/sigp/lighthouse/releases) page of the Lighthouse repository.
 
-## Pre-Built Electron Packages
+## Running the Docker container (Recommended)
 
-There are pre-compiled electron packages for each operating systems which can
-be downloaded and executed. These can be found on the
-[releases](https://github.com/sigp/siren/releases) page of the
-Siren repository.
+The most convenient way to run Siren is to use the Docker images built and published by Sigma Prime.
 
-Simply download the package specific to your operating system and run it.
+ They can be found on [Docker hub](https://hub.docker.com/r/sigp/siren/tags), or pulled directly with `docker pull sigp/siren`
 
-## Building From Source
-
-### Requirements
-
-Building from source requires `Node v18` and `yarn`.
-
-### Building From Source
-
-The electron app can be built from source by first cloning the repository and
-entering the directory:
-
-```
-git clone https://github.com/sigp/siren.git
-cd siren
-```
+Configuration is done through environment variables, the easiest way to get started is by copying `.env.example` to `.env` and editing the relevant sections (typically, this would at least include adding `BEACON_URL`, `VALIDATOR_URL` and `API_TOKEN`)
```

Comment:
The command to start Siren below does not include the password, so it may be good to add the password in the `.env` file.
```suggestion
Configuration is done through environment variables, the easiest way to get started is by copying `.env.example` to `.env` and editing the relevant sections (typically, this would at least include adding `BEACON_URL`, `VALIDATOR_URL`, `API_TOKEN` and `SESSION_PASSWORD`)
```